### ISSUE TITLE
Change master to main

### ISSUE THREAD

**Issue by ljhwang** [GeodynamicWorldBuilder/WorldBuilder#516]
Update the name of the central branch to main in accordance with best practices.

Some guidance:  https://github.com/github/renaming

**Comment by MFraters** [comment]
The central branch was already renamed to main (see #350). Where do you still see the use of master?

**Comment by ljhwang** [comment]
You are correct. I think I saw it on the github.io page and then have not renamed my branch! which i will do  right away ....
